tools/gcs-bucket-mover/README.md

```diff
@@ -0,0 +1,243 @@
+# GCS Bucket Mover Tool
+
+Currently there is no built in way to move a GCS bucket and all objects from one project to another. The manual process is:
+1. Note down all settings related to the source bucket
```

Comment:
Why are two copies required? Why not copy directly from the source bucket to the target bucket in the target project?

Reply:
Bucket names are globally unique, so you cannot create it in the target project until it has been deleted from the source project.

Reply:
Fair point! Thanks.